src/material-examples/chips-drag-and-drop/chips-drag-drop-example.ts

```diff
@@ -0,0 +1,31 @@
+import {Component} from '@angular/core';
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+
+export interface Vegetable {
+    name: string;
+}
+
+/**
+ * @title Chips Drag and Drop
+ */
+
+@Component({
+    selector: 'chips-drag-drop-example',
+    templateUrl: 'chips-drag-drop-example.html',
+    styleUrls: ['chips-drag-drop-example.css']
```

Comment:
This would feel a bit nicer with some animations. You can add the following to the example's CSS:

```css
.example-chip.cdk-drag-animating {
  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
}

.cdk-drop-list-dragging .example-chip {
  transition: transform 250ms cubic-bezier(0, 0, 0.2, 1);
}
```